src/hooks.server.ts

```diff
@@ -25,4 +24,61 @@ export const handle = SvelteKitAuth({
       },
     },
   ],
+  callbacks: {
```

Comment:
Do they have default behavior? If so, need to understand why it doesn't work with it.

Also need to make sure to apply it in all others apps that work with auth.js.

Reply:
We will need to add to NextAuth as well, it looks like this is not enabled by default. Good point @dorsha